bartop-ui/src/components/Auth/VerifyForm/VerifyForm.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Formik } from 'formik';
+
+import TextInput from '../../TextInput/TextInput';
+import strings from '../../../strings';
+
+import {
+  StyledForm,
+  PromptContainer,
+  Prompt,
+  InputContainer,
+  SubmitContainer,
+  SubmitButton
+} from '../StyledComponents';
+
+const MissingCodePrompt = styled.p`
+  font-size: small;
+  font-style: italic;
+
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
+const VerifyForm = ({ history, email, verifyCode }) => (
+  <Formik
+    initialValues={{
+      verificationCode: ''
+    }}
+    validate={({ verificationCode }) => ({})}
```

Comment:
why is this validator here?

Reply:
Because the `onSubmit` can imperatively set an error, I'm using the empty validator to clear any errors set from that API call. I'll check the `Formik` docs to see if there's a cleaner way to do this

Reply:
Cool. If you can find a cleaner way, that would be great, but if not, no worries. I would just add a comment explaining why that is there